src/main/java/core/basesyntax/ColorOfBall.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public enum ColorOfBall {
```

Comment:
```suggestion
public enum Color {
```

Not specifically for this task, but in general, we can use color not only for the ball. It's better to give generic names unless it's something very specific.